### DISCUSSION TITLE
How can I render a component inside a <Dialog> that can be positioned outside of <DialogContent>?

### DISCUSSION THREAD

**Discussion by jasonmng** [radix-ui/primitives#2650] [Help] [answered]
I'm trying to add a typeahead dropdown box, but currently the box is just extending the height of the dialog container and causing scrollbars to appear. I tried using a react portal to absolutely position an element outside of <Dialog> but it won't accept pointer events. How can I achieve adding a dropdown box like this?

**Comment by joaom00** [comment, marked as answer]
You can add `pointer-events: auto` to the dropdown when using with portal

**Reply by jasonmng** [reply]
Yes, thank you! that worked 🤦‍♂️
